lib/internal/bootstrap/realm.js

```diff
@@ -124,6 +124,7 @@ const legacyWrapperList = new SafeSet([
 // beginning with "internal/".
 // Modules that can only be imported via the node: scheme.
 const schemelessBlockList = new SafeSet([
+  'metrics',
```

Comment:
Since we will expose this API under perf_hooks, this doesn't seem necessary anymore